### DISCUSSION TITLE
Are non-numeric sub-addresses added at creation time considered to be keys?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#462] [Q&A] [answered]
Reading [`anonaddy.com/blog/add-recipients-to-aliases-at-creation-and-new-uuid-aliases`](https://anonaddy.com/blog/add-recipients-to-aliases-at-creation-and-new-uuid-aliases/), I realized that some of my addresses might be causing AnonAddy to unnecessary forward messages to addresses that don't exist. When creating a new address, I:

1. Get the current Epoch time.

1. Convert that to Base-36 for conciseness.

1. Add that as the address,

1. and do the same for the plus address (at creation time).

This is so that if I ever somehow forget to sub-address an address when posting it online, the contacter shall at least send its message to the default sub-address, which is slightly better than nothing (the address without a sub-address).

However, I don't want this to be in some way performing functionality that I don't expect. Can anyone confirm either way?

**Comment by willbrowningme** [comment, marked as answer]
No, you can see the `key` for each recipient on the [recipients](https://app.anonaddy.com/recipients) page.
